### PR TITLE
Updating mysql configuration to follow mysql container documentation

### PR DESCRIPTION
**Pull request by gabrielpreston**
To resolve issue brought up in https://github.com/laradock/laradock/issues/583

### CODE REVIEW

**Comment by appleboy** [comment]
Please also remove `startup` script.

**Comment by gabrielpreston** [comment]
@Mahmoudz I missed the comment on this PR while on vacation. Was this closed only because I hadn't followed up on the comment from @appleboy or for another reason?

**Comment by Mahmoudz** [comment]
@gabrielpreston I just noticed around 150 issues and over 20 PR today so I thought closing some issues especially old ones quickly just to clean up. If an issue still exist someone is gonna open it again or ask "like you" for reopening. And that's exactly what I'm gonna do. Thanks,

**Comment by Mahmoudz** [comment]
@gabrielpreston "sorry I left the wrong comment before".

I just closed it while closing a big number of Issues and PR's todays. Mainly old ones and those who seems pending for over a week (PR's in that case).

I'll reopen again that now. Thanks,

**Comment by gabrielpreston** [comment]
Removed the unused file and resolved the merge conflicts

File: docker-compose.yml [hunk 1]
```diff
@@ -148,11 +148,11 @@ services:
...
-            args:
-                - MYSQL_DATABASE=${MYSQL_DATABASE}
-                - MYSQL_USER=${MYSQL_USER}
-                - MYSQL_PASSWORD=${MYSQL_PASSWORD}
-                - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
+        environment:
+            MYSQL_DATABASE: homestead
+            MYSQL_USER: homestead
+            MYSQL_PASSWORD: secret
+            MYSQL_ROOT_PASSWORD: root
```
Comment (alchimik) [inline docker-compose.yml]: should be something like this:
```
        environment
            MYSQL_DATABASE: ${MYSQL_DATABASE}
            MYSQL_USER: ${MYSQL_USER}
            MYSQL_PASSWORD: ${MYSQL_PASSWORD}
            MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
```

see mariadb - https://github.com/laradock/laradock/blob/master/docker-compose.yml#L169

**Comment by jfoliveira** [comment]
For me it only worked using the YAML array notation, as below:

```yaml
    mysql:
        build: 
            context: ./mysql
        environment:
            - MYSQL_DATABASE=homestead
            - MYSQL_USER=homestead
            - MYSQL_PASSWORD=secret
            - MYSQL_ROOT_PASSWORD=root
```

**Comment by alchimik** [comment]
can be closed. see #758

**Pull request closed without merging**